src/main/java/Storage.java

```diff
@@ -1,10 +1,9 @@
-import java.io.File;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
+import java.io.*;
```

Comment:
Your code generally follows the coding standard as per usual, which is a good thing. However, please avoid importing all the files and only import those that you require for the purposes of your program. Overall good implementation which makes it easy to follow the code and read. Just have to implement the next few levels and you are good to go!